CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ Adds support for ATT&CK v11.
 ## Improvements
 
 - Revised the rigid versioning structure in the configuration file which enforced too many constraints on importing custom data. Added new fields `version.version` and `domain.identifier` to improve how Navigator derives domain information. More details are provided in the [integration documentation](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/develop/docs/integrations.md#attck-navigator-integration) on how to integrate the ATT&CK Navigator with the ATT&CK Workbench. See issue [#370](https://github.com/mitre-attack/attack-navigator/issues/370).
-- Fixed an issue where a long technique name or metadata value would overflow outside of the technique tooltip. See issue [#359](https://github.com/mitre-attack/attack-navigator/issues/359).
+- Fixed an issue where a long technique name or metadata value would overflow outside the technique tooltip. See issue [#359](https://github.com/mitre-attack/attack-navigator/issues/359).
+- Added a feature to pin a technique cell when viewed in the matrix, enabling the user to scroll the details of the technique in the tooltip. See issue [#320](https://github.com/mitre-attack/attack-navigator/issues/320).
```

Comment:
This should be moved up to a new changelog section for v4.6.1, since v4.6.0 has been released.